test/specs/scale.time.tests.js

```diff
@@ -297,8 +297,8 @@ describe('Time scale tests', function() {
 		var xScale = chart.scales.xScale0;
 
 		// Counts down because the lines are drawn top to bottom
-		expect(xScale.ticks[0].value).toBe('Jan 2');
-		expect(xScale.ticks[1].value).toBe('May 8');
+		expect(xScale.ticks[0].value).toEqualOneOf(['Jan 1', 'Jan 2']);
```

Comment:
Hmm, ok. I'm relatively sure I tried changing it to that earlier. At the moment it's passing for me with or without the change locally.

Anyway, I've updated the PR to used strings instead of timestamps